src/pages/iou/request/step/IOURequestStepParticipants.tsx

```diff
@@ -41,7 +41,9 @@ function IOURequestStepParticipants({
     const participants = transaction?.participants;
     const {translate} = useLocalize();
     const styles = useThemeStyles();
-    const selectedReportID = useRef<string>(reportID);
+
+    // We need to set selectedReportID if user has navigated back from confirmation page and navigates to confirmation page with already selected participant
+    const selectedReportID = useRef<string>(participants?.length === 1 ? participants[0]?.reportID ?? reportID : reportID);
```

Comment:
Bug was
1. User enters amount
2. Selects participants and clicks "Next"
3. Goes to confirmation page and see's Category, Tags and Tax fields
4. User goes back to 1 and edit's amount(doesn't matter)
5. User without selecting a participant clicks "Next" because participant was already selected(This is root cause)
6. Goes to confirmation page and doesn't see Category, Tags and Tax fields because reportID and policyID is not properly set from 5 above.

Let me know if you think this is not the correct way and should use separate hook or may be something else. Open to other solutions.